util.py

```diff
@@ -0,0 +1,349 @@
+"""This file defines the utils for the model.
+
+This file contains helper functions used by train_model.py. It defines
+function for creating the vocabulary and loading and preprocessing the 
+dataset. Additionally, it contains functions to compute certain
+metrics like F1 score, sentence-level semantic frame accuracy, etc.
+Some of the code was in this file was taken from 
+https://github.com/ZephyrChenzf/SF-ID-Network-For-NLU/blob/master/utils.py
```

Comment:
I think it is fine to create a fork. Please re-format this file to use the 2-character indents as in the other files?